### DISCUSSION TITLE
Possibility of adding user/device conditions to popups

### DISCUSSION THREAD

**Discussion by TinyShark** [Clooos/Bubble-Card#332] [Feature requests]
I was thinking of using bubble to display my alarm card based on a door opening trigger.

Problem is I would not want the popup to open on all devices that were currently on that view.

Is it possible to access the hass.user and implement a condition based on that?

e.g:

```
type: custom:bubble-pop-up
hash: '#alarmpanel'
name: Alarm
icon: mdi:siren
trigger_entity: binary_sensor.front_door
trigger_state: 'on'
trigger_close: true
users:
  - 581fca7fdc014b8b894519cc531f9a04
  - frontdooripad
```

**Comment by Clooos** [comment]
Hi! This is now possible in the new release! Check the changelog 😃 

https://github.com/Clooos/Bubble-Card/releases/tag/v2.3.1
